modules/networking/main.tf

```diff
@@ -0,0 +1,25 @@
+resource "google_compute_network" "vpc" {
+  name                    = "${var.namespace}-vpc"
+  description             = "Ctrlplane VPC Network"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "default" {
+  name          = "${var.namespace}-subnet"
+  ip_cidr_range = "10.10.0.0/16"
+  network       = google_compute_network.vpc.self_link
+}
+
+resource "google_compute_global_address" "private_ip_address" {
```

Comment:
turns out we actually need this so that the pod can talk to the cloudsql instance since it lives outside the VPC